servicemanager/include/python2.7

```diff
@@ -0,0 +1 @@
+/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
```

Comment:
Don't think we need this :stuck_out_tongue:

Reply:
You saw nothing